src/atmosphere.py

```diff
@@ -69,6 +69,17 @@ def calc_viscosity(self, altitude: float) -> float:
     return viscosity
 
 
+  def update_atmosphere(self, altitude: float) -> None:
```

Comment:
These are all called in the __init__, why do we need to update more than once per each state?

Reply:
Changed it to use update function in initializer.